<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sprite edit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QWidget" name="wEdit" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Motion</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbMotionType">
               <item>
                <property name="text">
                 <string>Nothing</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Walk</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Jump</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Climb</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Hit</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Bomb</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Current frame</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QCheckBox" name="cbHurt">
               <property name="text">
                <string>Hurt</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="wFrame" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Preview</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QWidget" name="wPreview" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbSpriteSheet">
      <property name="title">
       <string>Sprite sheet</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pbZoomPlus">
            <property name="text">
             <string>Zoom +</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbZoomMoins">
            <property name="text">
             <string>Zoom -</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>945</width>
            <height>438</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>977</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actNew"/>
    <addaction name="actOpen"/>
    <addaction name="separator"/>
    <addaction name="actSaveAs"/>
    <addaction name="actSave"/>
    <addaction name="separator"/>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menuSprite_sheet">
    <property name="title">
     <string>Sp&amp;rite sheet</string>
    </property>
    <addaction name="actSSopen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSprite_sheet"/>
  </widget>
  <action name="actNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actSaveAs">
   <property name="text">
    <string>&amp;Save as</string>
   </property>
  </action>
  <action name="actSave">
   <property name="text">
    <string>Sa&amp;ve</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actSSopen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
